src/settings.js

```diff
@@ -829,6 +829,12 @@ var MODULARIZE = 0;
 // (since you arean't creating the instance yourself).
 var MODULARIZE_INSTANCE = 0;
 
+// If we separate out asm.js with the --separate-asm option,
+// this is the name of the variable where the generated asm.js
+// Module is assigned to. This name can either be a property
+// of Module, or a freestanding variable name, like "var asmJsModule".
+var ASM_MODULE_NAME = 'Module["asm"]';
+
```

Comment:
this part could be a separate PR, I think?

Reply:
#7949